### ISSUE TITLE
Request: ME intrefaces accepting acceleration cards

### ISSUE THREAD

**Issue by akvgergo** [AppliedEnergistics/Applied-Energistics-2#1114]
If I use an interface for crafting, it only has a limited speed for outputing the required materials. Normally, this isn't an issue, but with enough machines and a system to handle the distribution between those machines, the speed at which the interface outputs becomes the slowest part of the crafting. The ability to speed this up would be nice.

**Comment by Aredrih** [comment]
As far as I know, the outputting speed of a interface depends on the speed of the corresponding crafting cpu. Adding more co processor unit should speed up the interface.

Also, cpu with the most co processor are the more likely to be choose for things like auto request so if you have block with crafting card, they will request those first.

Through, I'm not part of the ae2 staff so I might be wrong but that should work.

**Comment by thatsIch** [comment]
@Aredrih is right,

additionally you want to process stuff in parallel or pre-craft if you do too much stuff at once. If you put 2 patterns into 2 different interfaces, you will obviously double the through-put.
